### PR TITLE
stdlib: replace an obsolete initialize function in UnsafeBufferPointer.swapAt with the new one

### PR DESCRIPTION
**Pull request by eeckstein**
This brings back performance on stdlib sort because the obsoleted function in UnsafePointer is not inlinable.

### CODE REVIEW

**Comment by eeckstein** [comment]
@swift-ci smoke test

**Comment by eeckstein** [comment]
@swift-ci smoke benchmark

File: stdlib/public/core/UnsafeBufferPointer.swift.gyb [hunk 1]
```diff
@@ -361,7 +361,7 @@ extension Unsafe${Mutable}BufferPointer: ${Mutable}Collection, RandomAccessColle
     let pj = (_position! + j)
     let tmp = pi.move()
     pi.moveInitialize(from: pj, count: 1)
-    pj.initialize(to: tmp, count: 1)
+    pj.initialize(repeating: tmp, count: 1)
```
Comment (jrose-apple) [inline stdlib/public/core/UnsafeBufferPointer.swift.gyb]: I think the right fix would be `initialize(to: tmp)`.

Comment (eeckstein) [reply stdlib/public/core/UnsafeBufferPointer.swift.gyb]: thanks, I changed it

**Comment by eeckstein** [comment]
@swift-ci smoke test

**Review by airspeedswift — APPROVED** [review]

**Pull request merged**